src/node_crypto.cc

```diff
@@ -2006,66 +2074,18 @@ void SSLWrap<Base>::GetPeerCertificate(
   }
 
   // Clone `ssl_certs`, because we are going to destruct it
-  peer_certs = sk_X509_new(nullptr);
-  if (cert != nullptr)
-    sk_X509_push(peer_certs, cert);
-  for (int i = 0; i < sk_X509_num(ssl_certs); i++) {
-    cert = X509_dup(sk_X509_value(ssl_certs, i));
-    if (cert == nullptr)
-      goto done;
-    if (!sk_X509_push(peer_certs, cert))
-      goto done;
+  if (!CloneSSLCerts(&cert, &peer_certs, ssl_certs)) {
+    goto done;
   }
```

Comment:
Can you drop the braces for consistency with the surrounding code?

Reply:
Yeah, I'll remove them